.github/workflows/qns.yml

```
@@ -0,0 +1,47 @@
+name: qns # (quic-network-simulator)
+
+on:
+  push:
+    branches: ["main", "docker"] # TODO: remove docker
```

```suggestion
    branches: ["main"]
```

To be removed before merge.